docs/root/configuration/access_log.rst

```diff
@@ -29,19 +29,27 @@ are noted.
 
 The following command operators are supported:
 
+.. _config_access_log_format_start_time:
+
 %START_TIME%
   HTTP
     Request start time including milliseconds.
 
   TCP
     Downstream connection start time including milliseconds.
 
-  START_TIME can be customized using a `format string <http://en.cppreference.com/w/cpp/io/manip/put_time>`_, for example:
+  START_TIME can be customized using a `format string <http://en.cppreference.com/w/cpp/io/manip/put_time>`_.
+  In addition to that, START_TIME also accepts :code:`%f` to get the subsecond digits. The default is 9 digits (nanosecond).
+  The number of digits can be specified by specifiying the digit number after :code:`%`, e.g. :code:`%3f` to get 3 digits.
+
+  Examples of formatting START_TIME is as follows:
 
 .. code-block:: none
 
   %START_TIME(%Y/%m/%dT%H:%M:%S%z %s)%
 
+  %START_TIME(%s%3f)%
```

Comment:
Are we missing documentation here on `%s` (vs. `%S`) as a valid formatter? I don't see it listed at http://en.cppreference.com/w/cpp/io/manip/put_time. I gather this is the epoch time? Should there be a `.` seperator in these examples, e.g. `%s.%3f`?

Reply:
> Are we missing documentation here on %s (vs. %S) as a valid formatter?